components/Footer.tsx

```
@@ -1,9 +1,81 @@
   return (
-    <div>Footer</div>
+    <footer className="flexCenter mb-24">
+      <div className="padding-container max-container flex w-full flex-col gap-14">
+        <div className="flex flex-col items-start justify-center gap-[10%] md:flex-row">
+          <Link href="/" className="mb-10">
+            <Image src="hilink-logo.svg" alt="logo" width={74} height={29}/>
+          </Link>
+
+          <div className='flex flex-wrap gap-10 sm:justify-between md:flex-1'>
+            {FOOTER_LINKS.map((columns) => (
+              <FooterColumn title={columns.title}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)